hyperspy/drawing/mpl_he.py

```diff
@@ -20,10 +20,11 @@
 import logging
 
 from traits.api import Undefined
-
+import matplotlib as mpl
+from ipywidgets.widgets import HBox, VBox
```

Comment:
Move this import to where it is used and put it in a try, except block to handle the case, where ipywidgets is not installed.